pallets/loans/src/tests/market.rs

```diff
@@ -0,0 +1,187 @@
+use crate::{
+    mock::MARKET_MOCK,
+    mock::{Loans, Origin, Runtime, ALICE, DOT, NATIVE},
+    tests::ExtBuilder,
+    Error, InterestRateModel, Market, MarketState,
+};
+use frame_support::assert_noop;
+use primitives::{Rate, Ratio};
+use sp_runtime::FixedPointNumber;
+
+const PENDING_MARKET_MOCK: Market = {
+    let mut market = MARKET_MOCK;
+    market.state = MarketState::Pending;
+    market
+};
+
+macro_rules! rate_model_sanity_check {
```

Comment:
This macro was created to avoid duplicated code and is used to test rate model sanity checks of both `add_market` and `update_market` calls